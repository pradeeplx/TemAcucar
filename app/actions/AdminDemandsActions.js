import { apiAction } from './BasicActions'

export function list(credentials, currentUser, offset = 0) {
  return apiAction({
    prefix: 'ADMIN_DEMANDS_LIST',
    path: '/demands?filter=admin&offset=' + offset,
    credentials,
    currentUser: () => currentUser,
    processResponse: (response, json) => {
      return { list: json }
    },
  })
}
