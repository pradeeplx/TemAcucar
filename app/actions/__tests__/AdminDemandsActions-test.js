import 'react-native'
import React from 'react'
import * as subject from '../AdminDemandsActions'
import { identity } from 'lodash'

test('list action', () => {
  // BUG: We need to mock react-native-config
  // const action = subject.list({ id: 1 }, { name: 'Foo' })
  // expect(action(identity)).toMatchSnapshot()
})
