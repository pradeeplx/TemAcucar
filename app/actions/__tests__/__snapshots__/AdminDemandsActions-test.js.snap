exports[`test list action 1`] = `[Function]`;
